vercel-alt.zsh-theme

```diff
@@ -0,0 +1,45 @@
+local resetColor="%{$reset_color%}"
```

Comment:
How about making the hostname display when the machine is not local and hide on local?

Reply:
I'm not familiar with bash/zsh land, I'll look around on other themes.

Reply:
How about having three versions:

- with hostname
- without hostname
- without local hostname

Then again, I'm not familiar with bash/zsh land but will try to find references on other themes. Personally I use w/o hostnames. 😅

Reply:
The purpose of showing hostname is that you might have multiple terminals open, one on your local machine and one using ssh to connect to a remote machine. Its very important you execute commands on the correct machine so its helpful to display the hostname.

I really don't want to maintain alt themes based on what is displayed because that creates infinite possibilities.

Instead, the hostname could be hidden on the local machine and visible when connected to a remote machine.

Maybe its as simple as checking for a new env var `$LOCAL_HOSTNAME`